.devfile.yaml

```diff
@@ -180,15 +193,27 @@ commands:
           component: runtime
           commandLine: |
               CONFIG=${CONFIG:-default.yaml}
-              BOARD=${BOARD:-ti-06}
+              NAMESPACE=${NAMESPACE:-$(oc project -q)}
+              LEASE_RELEASE_TASK=$(oc -n ${NAMESPACE} get taskrun --sort-by=.metadata.creationTimestamp -o name | tail -1)
+              LEASE_ID=$(oc -n ${NAMESPACE} get taskrun.tekton.dev/${LEASE_RELEASE_TASK} -o jsonpath='{.spec.params[0].value}')
+              EXPORTER=$(oc get leases.jumpstarter.dev -A | grep ${LEASE_ID} | awk '{print $5}')
```

Comment:
might require more role addition, this is mainly for devspace test.
the CI will need a task using the current lease